WiserTaskScheduler/WiserTaskScheduler/Core/Services/ConfigurationsService.cs

```diff
@@ -304,7 +306,36 @@ private async Task<bool> SkipAction(JObject resultSets, ActionModel action)
                     return true;
                 }
             }
+            
+            if (!String.IsNullOrWhiteSpace(action.OnlyWithValue))
+            {
+                var parts = action.OnlyWithValue.Split(",");
 
+                try
+                {
+                    var defaultValue = String.Empty;
+                    var defaultValueIndex = parts[0].IndexOf('?');
+                    if (defaultValueIndex >= 0)
+                    {
+                        defaultValue = parts[0][defaultValueIndex..];
+                        parts[0] = parts[0][..defaultValueIndex];
+                    }
+                    
+                    var state = ReplacementHelper.GetValue($"{parts[0]}{defaultValue}", ReplacementHelper.EmptyRows, resultSets, false);
+                    
+                    if (state != parts[1])
```

Comment:
This assumes that that the `parts` array always has 2 values. The property `OnlyWithValue` seems to be a configuration/setting that users/developers have to setup. That means user errors will be possible and users maybe forget to add a comma to that value.

Can you add a check to your code to make sure that parts actually contains 2 values? If it doesn't, log a clear error or something. Otherwise we'll get exceptions that might be difficult to find/debug.